cmd/ci-op-configs-mirror/main.go

```diff
@@ -74,12 +75,23 @@ func main() {
 			rbc.CanonicalGoRepository = strP(fmt.Sprintf("github.com/%s/%s", repoInfo.Org, repoInfo.Repo))
 		}
 
-		repoInfo.Org = o.toOrg
+		if rbc.ReleaseTagConfiguration != nil {
+			filterReleaseTagConfiguration(rbc.ReleaseTagConfiguration)
```

Comment:
readability:

1. we don't really filter anything, that name is confusing. We are transforming, changing or generating something new, but not filtering.
2. it stutters (I admit it's hard to avoid here)

I'm also not a fan of abusing side effects (modifying the struct by passing the method a pointer). Ideally I'd like see something like this:

```go
rbc.ReleaseTagConfiguration = privateReleaseTag(rbc.ReleaseTagConfiguration)
...
```

Reply:
addressed